### ISSUE TITLE
Chroma integration won't work after latest udpate

### ISSUE THREAD

**Issue by farzbood** [deepset-ai/haystack-core-integrations#1136]
**Describe the bug**
The code example from the [repo](integrations/chroma/example) was working before the latest updates for the `haystack-ai` and `chroma-haystack` with both models (in-memory and persistent), but it just exit code execution at [indexing.run line](https://github.com/deepset-ai/haystack-core-integrations/blob/518cf27a8e765306bef4f6e0c7e1086169a11d4b/integrations/chroma/example/example.py#L24) with no output and/or error code! (log points show that in screenshots below)

**Terminal Output: (No Output/Error Code)**
![image](https://github.com/user-attachments/assets/f1a49d86-d85b-4420-a651-cd849546f663)

**Debug Console: (Execution never reach the next log point)**
![image](https://github.com/user-attachments/assets/c78922fd-78c6-4aa5-97df-c47a3489037b)

**To Reproduce**
Just run the example code in the repo.

**Describe your environment (please complete the following information):**
 - OS: [Windows10 x86_64]
 - Haystack version: haystack-ai (2.6.1)
 - Integration version: chroma-haystack (0.22.1)
 - chromadb (0.5.13)

**Comment by anakin87** [comment]
I cannot reproduce the error.

I tried the following on Ubuntu with Python 3.8 and Python 3.10.
- Create a new virtual environment
- `pip install chroma-haystack`
- copy the directory example from this repository
- `cd example`
- `python example.py`

I get the expected result:
```
{'file_path': '.../example/data/usr_51.txt'} 1.4255602359771729
{'file_path': '.../example/data/usr_41.txt'} 1.4427379369735718
{'file_path': '.../example/data/usr_29.txt'} 1.474898099899292
```

In our CI, we test the document store on Linux, Windows and MacOS.

Feel free to add more details if the error persists.

**Comment by farzbood** [comment]
Hi,
Sorry for late feedback, I've created a clean `.env` and install `chroma-haystack` but still got the same results.
So, tracing the code and turns out that the `hnswlib` in `chroma` crash the execution, opened an [issue](https://github.com/chroma-core/chroma/issues/2992#issue-2608415832) there and waiting for the response. will let you know about the results as soon as they respond.
PEACE V

**Comment by farzbood** [comment]
Hi,
FYI, This is the [latest status of the issue thread](https://github.com/chroma-core/chroma/issues/2513#issuecomment-2457546803) in `chromadb` repo.
BTW:
Since you won't publish any **release/tag** from the `haystack-core-integrations` repo, there's no way for us to get informed of updates by **watching** feature of `github`! is there any other way to get notified?

**Comment by farzbood** [comment]
Resolved by the [chromadb 0.5.23](https://github.com/chroma-core/chroma/releases/tag/0.5.23) release for me.
